### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+**Fixed bugs:**
+
+- GitLab MR: undefined method `\[\]' for nil:NilClass \(NoMethodError\) [\#187](https://github.com/voxpupuli/modulesync/issues/187)
+- msync fails with nilClass error [\#172](https://github.com/voxpupuli/modulesync/issues/172)
+- Fix NoMethodError for --pr option \(caused by `module_options = nil`\) [\#188](https://github.com/voxpupuli/modulesync/pull/188) ([bittner](https://github.com/bittner))
+- Allow empty module options in self.pr\(\) [\#186](https://github.com/voxpupuli/modulesync/pull/186) ([raphink](https://github.com/raphink))
+
+**Closed issues:**
+
+- Remove file namespaces from configuration [\#27](https://github.com/voxpupuli/modulesync/issues/27)
```
Comment (ekohl) [inline CHANGELOG.md:24]: The last updates suggest it was resolved a while back. Do we want to include this?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+
+**Implemented enhancements:**
+
+- Symbolize keys in managed\_modules except for module names [\#185](https://github.com/voxpupuli/modulesync/pull/185) ([raphink](https://github.com/raphink))
+
+**Fixed bugs:**
+
+- GitLab MR: undefined method `\[\]' for nil:NilClass \(NoMethodError\) [\#187](https://github.com/voxpupuli/modulesync/issues/187)
+- msync fails with nilClass error [\#172](https://github.com/voxpupuli/modulesync/issues/172)
+- Fix NoMethodError for --pr option \(caused by `module_options = nil`\) [\#188](https://github.com/voxpupuli/modulesync/pull/188) ([bittner](https://github.com/bittner))
```
Comment (ekohl) [inline CHANGELOG.md:19]: This also ended up implementing `--noop`

File: Gemfile [hunk 3]
```diff
@@ -4,3 +4,7 @@ gemspec
 
 gem 'cucumber', '< 3.0' if RUBY_VERSION < '2.1'
 gem 'octokit', '~> 4.9'
+
+group :release do
+  gem 'github_changelog_generator',  :require => false, :git => 'https://github.com/voxpupuli/github-changelog-generator', :branch => 'voxpupuli_essential_fixes'
```
Comment (ekohl) [inline Gemfile:9]: This breaks the build on older Ruby versions.

Comment (bastelfreak) [reply Gemfile:9]: ups, fixed it

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,25 @@
 # Changelog
 
-## 2020-07-03 - 1.3.0
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
```
Comment (ekohl) [inline CHANGELOG.md:5]: This is irrelevant for a gem.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,25 @@
 # Changelog
 
-## 2020-07-03 - 1.3.0
+All notable changes to this project will be documented in this file.
```
Comment (ekohl) [inline CHANGELOG.md:3]: I think this speaks for itself.

File: CHANGELOG.md (hunk 4 above)
Comment (bastelfreak) [reply CHANGELOG.md:5]: updated it

**Comment by bastelfreak** [comment]
@ekohl please review again :)

**Pull request merged**